sync/etcd/etcd.go

```diff
@@ -61,7 +61,7 @@ func (s *Sync) Update(key, jsonString string) error {
 }
 
 //Delete sync update sync
-func (s *Sync) Delete(key string) error {
+func (s *Sync) Delete(key string, prefix bool) error {
 	s.etcdClient.Delete(key, false)
```

Comment:
prefix is ignored. I think it's safe to pass it to etcdClient.Delete(). if that's not the case, I'd add
```
if prefix {
    panic("Prefix not supported in ETCD v2")
}
```